Model/RefundFailedProcessor.php

```diff
@@ -56,12 +72,9 @@ public function processFailedWebhook($params, $jobName)
         $memoId = $this->collection->addFieldToFilter('refund_id', ['eq' => $refundId])
             ->getFirstItem()->getCreditMemoId();
         if ($memoId) {
-            $creditmemoModel = $this->creditmemoFactory->create();
-            $creditmemoModel->setData([
-                'entity_id' => $memoId,
-                'creditmemo_status' => 1
-            ]);
-            $creditmemoModel->save();
+            $memoData = $this->creditmemoRepository->get($memoId);
+            $memoData->addData(['state' => 1]);
```

Comment:
can we represent state in textual value?

Reply:
Updated